rules/Php82/Rector/Param/AddSensitiveParameterAttributeRector.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Php82\Rector\Param;
+
+use PhpParser\Node;
+use Rector\Core\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\ValueObject\PhpVersionFeature;
+use Rector\VersionBonding\Contract\MinPhpVersionInterface;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use Webmozart\Assert\Assert;
+
+/**
+ * @see \Rector\Tests\Php82\Rector\Param\AddSensitiveParameterAttributeRector\AddSensitiveParameterAttributeRectorTest
+ */
+class AddSensitiveParameterAttributeRector extends AbstractRector implements ConfigurableRectorInterface, MinPhpVersionInterface
+{
+    public const SENSITIVE_PARAMETERS = 'sensitive_parameters';
+
+    /**
+     * @var string[]
+     */
+    private array $sensitiveParameters;
+
+    /**
+     * @param mixed[] $configuration
+     */
+    public function configure(array $configuration): void
+    {
+        Assert::allString($configuration[self::SENSITIVE_PARAMETERS] ?? []);
+        $this->sensitiveParameters = (array) ($configuration[self::SENSITIVE_PARAMETERS] ?? []);
+    }
+
+    public function getNodeTypes(): array
+    {
+        return [Node\Param::class];
+    }
+
+    /**
+     * @param Node\Param $node
+     */
+    public function refactor(Node $node): ?Node\Param
+    {
+        if (! $this->isNames($node, $this->sensitiveParameters)) {
+            return null;
+        }
+
```

Comment:
check attribute exists should be here to avoid re-add on next run:

```php
        if ($this->phpAttributeAnalyzer->hasPhpAttribute($node, 'SensitiveParameter')) {
            return true;
        }
```